02-express-tutorial/app.js

```diff
@@ -3,18 +3,48 @@ console.log("Express Tutorial");
 //                          Express
 
 const Express = require("express");
+const cookieParser = require("cookie-parser");
 const app = Express();
 const { products } = require("./data");
+const { people } = require("./data");
 
-app.use(Express.static("./public"));
+const logger = require("./logger"); //week 3
+const peopleRouter = require("./routes/people");
+const authRouter = require("./routes/auth");
+const auth = require("./auth");
```

Comment:
To get full use of this auth middleware, you'll want to add it to the app with `app.use`. If you add this use statement before the `authRouter` below, you should see the expected behavior on log on and out